lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -804,6 +884,35 @@ module Resources
             'relationship': [
               {
                 'id': 'CI-59302320-0-0',
+                'extension': [
```

Comment:
Same question as the read patient access example.

Reply:
removed here https://github.com/cerner/fhir.cerner.com/pull/624/commits/95b78bfc03c61933a3a7b57577a122f4bf27dae5#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL805-L1207